robot/client/client_test.go

```diff
@@ -722,6 +766,12 @@ func TestClientRefresh(t *testing.T) {
 			callCountNames++
 			return emptyResources
 		}
+		injectRobot.MachineStatusFunc = func(ctx context.Context) (robot.MachineStatus, error) {
+			mu.Lock()
+			defer mu.Unlock()
+			callCountNames++
```

Comment:
Curious if you could use a different variable here. Is it right to assume that the `ResourceNamesFunc` above is not going to get called at all anymore?

Reply:
ugh - sorry missed this due to large-file folding. yes good call, will make separate counters for each and test accordingly

Reply:
ended up dropping a lot of useless code - thanks for catching!